app/frontend/components.js

```diff
@@ -51,28 +51,30 @@ window.mountFundraiser = (root: string, initialState?: any = {})  => {
   }
 };
 
-type callToolInitialState = {
+type callToolProps = {
   locale: string;
   title?: string;
   targets: any[];
   targetCountries: any[];
   countriesPhoneCodes: any[];
   pageId: string | number;
+  targetByCountryEnabled: boolean;
 };
 
-window.mountCallTool = (root: string, props: callToolInitialState) => {
+window.mountCallTool = (root: string, props: callToolProps) => {
   props = camelizeKeys(props);
 
   render(
     <ComponentWrapper locale={props.locale}>
       <CallToolView
         title={props.title}
         targets={props.targets}
-        targetCountries={props.targetCountries}
+        countries={props.countries}
         pageId={props.pageId}
         onSuccess={props.onSuccess}
         countriesPhoneCodes={props.countriesPhoneCodes}
-        targetCountryCode={props.targetCountryCode} />
+        countryCode={props.countryCode} />
+        targetByCountryEnabled={props.targetByCountryEnabled}
```

Comment:
Since all you're basically passing all props through, you can do: `<CallToolView {...props} />` - saves you time whenever you add new props.